nimsuggest/tester.nim

```diff
@@ -345,6 +344,12 @@ proc runTest(filename: string): int =
   result = report.len
 
 proc main() =
+  let nimsug = "bin" / addFileExt("nimsuggest_testing", ExeExt)
+  if not nimsug.fileExists():
+    const args = "c -o:bin/nimsuggest_testing -d:release nimsuggest/nimsuggest"
+    discard execShellCmd(getCurrentCompilerExe() & " " & args)
+
+  doAssert nimsug.fileExists, nimsug
```

Comment:
this can pass from a prior `nimsuggest` existing, I would put a check for a `0` exit code for the compile step above.